packages/translate/lib/__tests__/testLineResult.test.tsx

```diff
@@ -972,4 +972,966 @@ describe('Test line results translation', () => {
 		});
 		// ---- end of "translate "assert network"" ----
 	});
+
+	describe('Translate assert lines with then fail', () => {
```

Comment:
Just curios, was it copied from somewhere or did you generate all those examples on your own?

Reply:
I used same tests from testLineResult.test.tsx but with "then fail" line